### PR TITLE
Update CHANGELOG for 0.17.8

### PR DESCRIPTION
**Pull request by metral**

### CODE REVIEW

**Comment by chrsmith** [comment]
I don't know if it is worth mentioning in the changelog, but with this release we will start also deploying/publishing the `pulumi/actions` container on DockerHub, which is used for GitHub Actions. (https://github.com/pulumi/pulumi/pull/2646)

When we do publish the release, I'll need to confirm that the new container was build/published correctly though.

**Comment by metral** [comment]
@chrsmith i've updated the changelog to include https://github.com/pulumi/pulumi/pull/2646

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+
+## 0.17.8 (Released April 23, 2019)
 
 ### Improvements
 
 - Add a new `ignoreChanges` option to resource options to allow specifying a list of properties to
   ignore for purposes of updates or replacements.  [#2657](https://github.com/pulumi/pulumi/pull/2657)
 - Fix an engine bug that could lead to incorrect interpretation of the previous state of a resource leading to 
   unexpected Update, Replace or Delete operations being scheduled. [#2650]https://github.com/pulumi/pulumi/issues/2650)
+- Build/push pulumi/actions container with new SDK releases [#2646](https://github.com/pulumi/pulumi/pull/2646)
```
Comment (lukehoban) [inline CHANGELOG.md]: Total not - but perhaps escape `pulumi/actions` and/or link to docker hub?

**Comment by metral** [comment]
@lukehoban feedback addressed 👍

**Pull request merged**